glide.yaml

```diff
@@ -11,7 +11,7 @@ import:
 - package: github.com/gorilla/mux
   version: ~1.1.0
 - package: github.com/prometheus/client_golang
-  version: ~0.8.0
+  version: master
```

Comment:
IIRC, we decided on `master` as a default, which makes a lot of sense to me. We already have 

```
- package: github.com/spf13/cobra
  version: master
- package: github.com/spf13/pflag
  version: master
- package: github.com/spf13/viper
  version: master
```

some other external dependencies on master. Up for dicussion, fair.